WalletWasabi.Gui/Controls/WalletExplorer/WalletExplorerView.xaml

```diff
@@ -18,6 +18,15 @@
         </Animation>
       </Style.Animations>
     </Style>
+    <Style Selector="StackPanel.TreeViewRoot">
+      <Setter Property="Background" Value="Transparent" />
+      <Setter Property="Orientation" Value="Horizontal" />
+      <Setter Property="Spacing" Value="6" />
+      <Setter Property="Height" Value="20" />
+    </Style>
+    <Style Selector="StackPanel.TreeViewRoot > :is(Control)">
```

Comment:
Is this legit? :) What is this?

Reply:
Yes it's legit :D basically the ` > ` indicates direct descendant of the prior control then the `:is` operator checks if the target is a derivative of the class in its parameter. Docs for style selectors: http://avaloniaui.net/docs/styles/selectors#is